### PR TITLE
expand pronoun explanation and add resources

### LINKED ISSUE #1222: How about adding resources on the proper use of pronouns?
In the first episode "[Welcome](https://carpentries.github.io/instructor-training/01-welcome/index.html)", there is a box "Pronouns and Names" highlighting the importance of using correct names and pronouns to setting a tone of respect. 

I agree but failed to implement it properly. I asked COC and Malvika sent the following resources which I believe would be useful to everyone so I think we could add links in The Carpentries instructor training material:
- The proper use of pronouns in language: [https://lgbt.ucsf.edu/pronounsmatter](https://lgbt.ucsf.edu/pronounsmatter) 
- The importance of using pronouns: [https://www.mypronouns.org/](https://www.mypronouns.org/) 
- How to use personal pronouns: https://www.mypronouns.org/how (as a non-native speaker, I find it important to have examples of usage)
- How to deal with situations when you use the wrong pronoun: https://www.mypronouns.org/mistakes

### PR DESCRIPTION
**Pull request by brownsarahm**
This uses text mostly from a [PR to Data lessons](https://github.com/data-lessons/instructor-training/pull/111) @pachadotdev and should close #1222 

It also relates to #1409

### CODE REVIEW

**Comment by ndporter** [comment]
Checked and updated to fix one typo and add using preferred names (as well as pronouns) - to avoid deadnames etc. Awaiting review of changes before merging this and closing #1222 .

File: _episodes/01-welcome.md [hunk 1]
```diff
@@ -29,6 +29,18 @@ keypoints:
 > 
 > Note that pronouns are personal and some participants might prefer not to share them.
 > Do not force people to share their pronouns.
+> One reason to avoid pressuring people to share them is to allow people to share their 
+> gender identity only when they feel ready to. 
+> It is however necessary for all participants to use the provided pronouns and names when available.
```
Comment (karenword) [inline _episodes/01-welcome.md:34]:
```suggestion
> It is, however, necessary for all participants to use pronouns and names as listed when participants provide them. 
```

**Review by karenword — APPROVED** [review]
Minor suggestion to clarify language; address as you see fit!

**Comment by karenword** [comment]
In the prior update, one aim was to reduce the number of links to outside resources, to ward against link rot and improve control over maintenance. I'm ok with adding these for now, but we should consider building our own resources in this area.

**Comment by brownsarahm** [comment]
I liked @karenword's revision and opened an issue related to the point about links.

**Pull request merged**